com.b2international.snomed.ecl/src/com/b2international/snomed/ecl/Ecl.xtext

```diff
@@ -213,8 +213,10 @@ PropertyFilter:
 MemberFieldFilter:
 	refsetFieldName=UnquotedString comparison=Comparison;
 
-IdFilter:
-	ID_KEYWORD op=NON_NUMERIC_OPERATOR (ids+=Identifier | ROUND_OPEN (ids+=Identifier)+ ROUND_CLOSE);
+IdFilter
+	: ID_KEYWORD op=NON_NUMERIC_OPERATOR (ids+=Identifier | ROUND_OPEN (ids+=Identifier)+ ROUND_CLOSE)
```

Comment:
There is an overlap here and the API will be inconsistent with this change as there will be two ways to get the assigned ID(s) from the IdFilter, one for the regular ECL way and one for the new regex way.
Please change the construct to this instead:

```
    ID_KEYWORD op=NON_NUMERIC_OPERATOR ((searchType = EXACT_KEYWORD | REGEX_KEYWORD)? ids+=Identifier | ROUND_OPEN (ids+=Identifier)+ ROUND_CLOSE)
```

The default is the `exact` searchType if not defined. This construct will also allow future expansions of the search behavior and it is consistent with the other search type declaration methods.